nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -88,7 +88,7 @@ let
   #     result in incorrect boot entries.
 
   baseIsolinuxCfg = ''
-    SERIAL 0 38400
+    SERIAL 0 115200
```

Comment:
I don't know if this has some goofy side effects, but +1 -- I've made this patch on my own before :)